cryptol-saw-core/src/CryptolSAWCore/Cryptol.hs

```diff
@@ -950,6 +950,10 @@ prelPrims =
                                   --   (fin first, fin stride, stride >= 1, first >= bound, Literal first a) =>
                                   --   [(first - bound)/^stride]a
 
+    -- GF2 Polynomial primitives
+  , ("pmult",        flip scGlobalDef "Cryptol.ecPmult")     -- {u, v} (fin u, fin v) => [1 + u] -> [1 + v] -> [1 + (u + v)]
+  , ("pmod",         flip scGlobalDef "Cryptol.ecPmod")      -- {u, v} (fin u, fin v) => [u] -> [1 + v] -> [v]
```

Comment:
Is it worth opening an issue about filling in a definition for `pdiv` later?

Reply:
Are there any crypto algorithms that use `pdiv`? `git grep pdiv` in the `cryptol-specs` repo turns up nothing. I think the `Cryptol::Reference` implementation of `pdiv` is as good as we'll ever need.